vscode/open/command.py

```diff
@@ -215,32 +216,3 @@ def command(shell, args: list):
     def complete(shell, word, line):
         return []
 
-
-def _run_cmd(
-        cmd, get_output=False, print_output=False, suppress_errors=False, shell=False, return_exitcode=False
-):
-    if shell and isinstance(cmd, (list, tuple)):
-        cmd = " ".join([str(s) for s in cmd])
-    dtslogger.debug("$ %s" % cmd)
-    if get_output:
-        proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=shell)
-        proc.wait()
-        if proc.returncode != 0:
-            if not suppress_errors:
-                msg = "The command {} returned exit code {}".format(cmd, proc.returncode)
-                dtslogger.error(msg)
-                raise RuntimeError(msg)
-        out = proc.stdout.read().decode("utf-8").rstrip()
-        if print_output:
-            print(out)
-        return out
-    else:
-        if return_exitcode:
-            res = subprocess.run(cmd, shell=shell)
-            return res.returncode
-        else:
-            try:
-                subprocess.check_call(cmd, shell=shell)
-            except subprocess.CalledProcessError as e:
-                if not suppress_errors:
-                    raise e
```

Comment:
There are several of these `_run_cmd` functions scattered throughout various commands, we should unify them.